src/fund.jl

```diff
@@ -49,7 +49,7 @@ function getfund(; nsteps = default_nsteps, datadir = default_datadir, params =
     # ---------------------------------------------
 
     if params == nothing
-        parameters = loadparameters(datadir)
+        parameters = load_default_parameters(datadir)
     else
         parameters = params
     end
```

Comment:
[Trying this new suggestion feature...]
```suggestion
parameters = params === nothing ? load_default_parameters(datadir) : params
    end
```